src/components/hero.jsx

```diff
@@ -49,8 +49,8 @@ const Hero = () => {
 
       <Form />
 
-      <section className="block justify-center  items-center md:flex">
-        <div className="md:w-1/2 md:pr-20 md:text-left text-center">
+      <section className="block justify-center  items-center md:flex md:flex-col-reverse xl:flex xl:flex-row lg:flex lg:flex-row">
```

Comment:
Apologies for not having caught this before, `lg` applies classes to all large screens and above. So it will include `xl`. Adding `lg` and `xl` is a  bit redundant I think.
 ```suggestion
      <section className="block justify-center  items-center md:flex md:flex-col-reverse  lg:flex lg:flex-row">
```